pyvista/plotting/utilities/sphinx_gallery.py

```diff
@@ -6,9 +6,9 @@
 import pyvista
 
 BUILDING_GALLERY_ERROR_MSG = (
-    'pyvista.BUILDING_GALLERY must be set to True in your conf.py to capture '
-    'images within sphinx_gallery or when building documentation using the '
-    'pyvista-plot directive.'
+    "pyvista.BUILDING_GALLERY must be set to True in your conf.py to capture "
+    "images within sphinx_gallery or when building documentation using the "
```

Comment:
Is this true about the `pyvista-plot` directive requiring `pyvista.BUILDING_GALLERY is True`?  I didn't use it and it has been working.

Reply:
This requirement isn't listed at https://docs.pyvista.org/version/stable/extras/plot_directive.html

Reply:
This is only for sphinx-gallery usage

Reply:
`plot_directive` doesn't need this.

Reply:
I suspect your project is setting `pyvista.BUILDING_GALLERY = True` which happens to kick in with the `pyvista-plot` directive causing the `ImportError` when trying to run `export_vtksz`

Reply:
This isn't being set in the `conf.py` here https://github.com/MatthewFlamm/pyransame/blob/main/doc/conf.py

Reply:
Huh. Odd. Okay I will clone this and reproduce.

Reply:
Found the problem. BUILDING_GALLERY is hard set when using the `pyvista-plot` directive:

https://github.com/pyvista/pyvista/blob/7000c660e982e0279ed8d30111aa83d2ced4d0c0/pyvista/ext/plot_directive.py#L119-L120

Reply:
Then this comment here is inaccurate since it says it has to be in your `conf.py`.  Since this is a patch, maybe it can be a separate PR, although you did go through and change style here already.

Reply:
Yeah, this could use some cleanup. Going to save for later since this patch seems pretty necessary